CHANGELOG.asciidoc

```diff
@@ -13,9 +13,12 @@ https://github.com/elastic/beats/compare/v5.0.0-alpha1...master[Check the HEAD d
 ==== Breaking changes
 
 *Affecting all Beats*
+
 - Add scripts for managing the dashboards of a single Beat {pull}1359[1359]
+- On DEB/RPM installations, the binary files are now found under `/usr/share/{{beat_name}}/bin`, not in `/usr/bin`. {pull}1385[1385]
```

Comment:
I added that as a TODO in #1371, I'd prefer getting this one in quicker so we can test it with the nightlies.